superduper/components/vector_index.py

```diff
@@ -221,20 +221,33 @@ def trigger_ids(self, query: Query, primary_ids: t.Sequence):
         )
         return ids
 
+    def _create_predict_job(self, db, callable, deps, ids, job_id=None):
+        job = FunctionJob(
+            callable=callable,
+            args=[],
+            kwargs=dict(
+                vector_index=self.identifier,
+                ids=ids,
+                query=db[self.indexing_listener.outputs].dict().encode(),
+            ),
+        )
+        job(db=db, dependencies=deps)
+        return [job]
+
     @override
     def run_jobs(
         self,
         db: Datalayer,
         dependencies: t.Sequence[str] = (),
-        ids: t.List = [],
+        events: t.List = [],
         overwrite: bool = False,
         event_type: str = DBEvent.insert,
     ) -> t.Sequence[t.Any]:
         """Run jobs for the vector index.
 
         :param db: The DB instance to process
         :param dependencies: A list of dependencies
-        :param ids: List of ids.
+        :param events: List of events.
```

Comment:
Difference between `run_jobs` and `schedule_jobs` not clear.

Reply:
schedule_jobs puts events into queue
run_jobs runs the events consumed form the queue